spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -42,6 +42,9 @@
       it 'returns appropriate response' do
         get api("/clew/appliance_instances", user)
         expect(response.status).to eq 200
+
+        appl.reload
```

Comment:
To make `appl` "notice" the changes introduced by `initial_billing` in the appliance's deployment object. If you know a better way to fix it then I'm open to suggestions - moving `initial_billing` from `after_create` to `after_build` still does not update the in-memory state of `appl`. :(